WKData/Sources/WKData/Service/WKBasicService.swift

```diff
@@ -0,0 +1,223 @@
+import Foundation
+
+/// Use this service for the most basic networking service calls. It does not handle authentication.
+public final class WKBasicService: WKService {
```

Comment:
Are you intending `WKService` to serve some more general purpose than effectively performing network requests? If not, `WKNetworkService`, `WKBasicNetworkService`, and `WKNetworkServiceEnvironment` read as far more precise type identifiers to me.

Reply:
I can certainly appreciate the desire to keep it short (`WKService` over `WKNetworkService`) but from my first read of this work it didn't quite click for me. I think I mentally expect `Service` as something more generalized, and there being specializations of it – like what if there's some local data service, or some string localization service? That's just my personal take!

For time sake let's leave this as-is and pick this conversation back up in the future as `WKData` expands in functionality – sound good?